### DISCUSSION TITLE
missing in docs

### DISCUSSION THREAD

**Discussion by bitlab-code** [Milkdown/milkdown#1135] [Q&A]
Hello, I'm try to add a simple toolbar with "Heading1" "paragraph" and "link" buttons
but i can't find any info into the docs.

I'm started from online official example but I'm able to add only a "bold" feat.
Can anyone please help me? Thanks

https://stackblitz.com/edit/github-wkjzgg

**Comment by ahamelers** [comment]
I used the playground editor as a starting point to figure out creating a toolbar

https://github.com/Milkdown/website/blob/main/src/components/playground-editor/index.tsx
